.bbtravis.yml

```diff
@@ -181,6 +181,12 @@ script:
       export PATH=/tmp/bbvenv/bin/:$PATH
       isort --check -df `git ls-files |grep '.py$'`
```

Comment:
We can also replace pylint/isort/flake8 with ruff as it integrates the same rulesets.
Than we no longer have competing linters.

Reply:
started in https://github.com/buildbot/buildbot/pull/7326